subprojects/docs/src/docs/userguide/authoring-builds/working_with_files.adoc

```diff
@@ -436,17 +436,18 @@ include::sample[dir="snippets/files/fileTrees/kotlin",files="build.gradle.kts[ta
 
 You can see more examples of supported patterns in the API docs for link:{javadocPath}/org/gradle/api/tasks/util/PatternFilterable.html[PatternFilterable]. Also, see the API documentation for `fileTree()` to see what types you can pass as the base directory.
 
-By default, `fileTree()` returns a `FileTree` instance that applies some default exclusion patterns for convenience — the same defaults as Ant in fact. For the complete default exclusion list, see http://ant.apache.org/manual/dirtasks.html#defaultexcludes[the Ant manual].
+By default, `fileTree()` returns a `FileTree` instance that applies some default exclude patterns for convenience — the same defaults as Ant in fact. For the complete default exclude list, see http://ant.apache.org/manual/dirtasks.html#defaultexcludes[the Ant manual].
 
-If those default exclusions prove problematic, you can workaround the issue by using the {antManual}/Tasks/defaultexcludes.html[`defaultexcludes` Ant task], as demonstrated in this example:
+[[sec:change_ant_excludes]]
+If those default excludes prove problematic, you can workaround the issue by changing the default excludes in settings:
 
-.Changing Ant default exclusions for a copy task
+.Changing default excludes in settings
```

Comment:
Because we are changing the default excludes for _Gradle_. These default excludes happen to be the same as the ones for Ant.

Reply:
I'm not sure about this. We also break any links pointing at the `ant` ID by this change. And the user is changing the Ant excludes, Gradle simply takes them into account. (For example, if there was an Ant task executed, it would also take the configured excludes into account, no?)

I think we should keep talking about Ant excludes, and maybe clarify that we take them into account.

Reply:
I introduced the `ant` id, so there shouldn't be any breakages.

The documentation did not talk about Ant excludes, but default excludes. If we want to use Ant in the name, we would need to say Ant's default excludes. And it happens so that Gradle uses Ant's default excludes as default excludes. This is why I prefer using default excludes without mentioning Ant.